src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -186,6 +186,8 @@ protected override bool OnShutdown(int timeoutMilliseconds)
             this.shutdownDrainTarget = this.circularBuffer.AddedCount;
             this.shutdownTrigger.Set();
 
+            OpenTelemetrySdkEventSource.Log.DroppedExportProcessorItems(this.GetType().Name, this.droppedCount);
```

Comment:
this is not the exporter name. this'd be the name of the batchexportprocessor. Exporter example is "OTLPExporter", "ZipkinExporter" etc.

Reply:
"BatchExportProcessor<Activity> exporting to OTLP/Jaeger/Zipkin/ExporterName dropped N items due to buffer full." -- ~~ this is more helpful